ssh/build.yaml

```diff
@@ -9,5 +9,5 @@ codenotary:
   base_image: notary@home-assistant.io
 args:
   CLI_VERSION: 4.14.0
-  LIBWEBSOCKETS_VERSION: 4.1.4
-  TTYD_VERSION: 1.6.3
+  LIBWEBSOCKETS_VERSION: 4.2.1
+  TTYD_VERSION: 62a5b635c6718109c64a6d629014f2784959f02c
```

Comment:
What's the reason for this old commit?
Later commits in the repo fixes issues with webgl2

Reply:
@ludeeus Happy to go with a later commit, it looks like the community version of the ssh addon has moved forward to `3e37e33b1cd927ae8f25cfbcf0da268723b6d230`. I'll move this PR up as well.